### PR TITLE
[Models] [Relationships] AWS RDS Controller

### PR DESCRIPTION
**Pull request by Jougan-0**
**Notes for Reviewers**

This PR fixes #

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: server/meshmodel/aws-rds-controller/v1.4.3/v1.0.0/relationships/dbCluster-to-dbClusterParameterGroup.json [hunk 1]
```diff
@@ -0,0 +1,56 @@
+{
+    "schemaVersion": "relationships.meshery.io/v1alpha3",
+    "version": "v1.0.0",
+    "kind": "edge",
+    "type": "binding",
+    "subType": "configuration",
```
Comment (leecalcote) [inline server/meshmodel/aws-rds-controller/v1.4.3/v1.0.0/relationships/dbCluster-to-dbClusterParameterGroup.json:6]: @Jougan-0 is this an existing subtype? What visualization is associated with "configuration"?

File: server/meshmodel/aws-rds-controller/v1.4.3/v1.0.0/relationships/dbClusterSnapshot-to-dbCluster.json [hunk 2]
```diff
@@ -0,0 +1,56 @@
+{
+    "schemaVersion": "relationships.meshery.io/v1alpha3",
+    "version": "v1.0.0",
+    "kind": "hierarchical",
+    "type": "sibling",
+    "subType": "backup",
```
Comment (Jougan-0) [inline server/meshmodel/aws-rds-controller/v1.4.3/v1.0.0/relationships/dbClusterSnapshot-to-dbCluster.json:6]: Here too it says backup

File: server/meshmodel/aws-rds-controller/v1.4.3/v1.0.0/relationships/dbCluster-to-dbClusterParameterGroup.json (hunk 1 above)
Comment (Jougan-0) [reply server/meshmodel/aws-rds-controller/v1.4.3/v1.0.0/relationships/dbCluster-to-dbClusterParameterGroup.json:6]: I don't know how did it took configuration and backup. I'll investigate why did the LLM gave this.

File: server/meshmodel/aws-rds-controller/v1.4.3/v1.0.0/relationships/dbClusterSnapshot-to-dbCluster.json (hunk 2 above)
Comment (leecalcote) [reply server/meshmodel/aws-rds-controller/v1.4.3/v1.0.0/relationships/dbClusterSnapshot-to-dbCluster.json:6]: While these aren't specifically supported, I don't think they hurt anything either (or they shouldn't, that is).

**Review by leecalcote — APPROVED** [review]

**Pull request merged**